.github/labeler.yml

```diff
@@ -0,0 +1,10 @@
+# changes to documentation
+"area/documentation": doc/**/*
+
+# changes to the core lib package
+"area/lib": ./*.go
```

Comment:
I think that this should include `doc/**`. At the same time an "area/cli" category can be created for `cobra/**`.